contracts/test/contract.SL2RD.test.js

```diff
@@ -118,6 +119,26 @@ contract("SL2RD", (accounts) => {
         await splitContract.countAllocatedCommunitySlots(),
         1
       );
+
+      const newDistribution = [
+        accounts[1],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+        accounts[0],
+      ];
+
+      for (let i = 0; i < ownerAddresses.length; i++) {
+        assert.equal(
+          (await splitContract.currentSplitDistributionTable())[i],
```

Comment:
Lets call this once and iterate over the resulting array rather than re-reading the table on each iteration.